app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/scanning.rb

```diff
@@ -22,8 +22,6 @@ def perform_metadata_scan(ost)
     $log.debug "#{log_pref} VM = #{vm_name}"
 
     args1 = ost.args[1]
-    args1['ems'][:use_vim_broker]     = MiqServer.use_broker_for_embedded_proxy?(args1['ems']['connect_to'])
-    args1['ems'][:vim_broker_drb_uri] = MiqVimBrokerWorker.drb_uri if args1['ems'][:use_vim_broker]
```

Comment:
Does the drb_uri column go away now?

Reply:
On the miq_workers table yeah it probably can